dcos/dcos-ui
# DCOS-55633: Group quota detail page

Quota detail page with cards.

Closes https://jira.mesosphere.com/browse/DCOS-55633

## Testing
0. Start localhost and login.
1. Update your `Config.dev.ts`
```
uiConfiguration: {
...
  features: {
    quota: true
  }
},
  useFixtures: true,
  ...
```
2. Go to services page.
3. Open quota tab.
4. Click on a group.
5. Verify that the detailed view for the groups is shown and it looks like the design from the jira.
5.5 (Optional) Test it on smaller window to make sure it looks okay.
6. Verify that the added tests make sense.

## Trade-offs
None.

## Dependencies
https://github.com/dcos/dcos-ui/pull/3977

## Screenshots
![Screenshot from 2019-07-04 14-18-26](https://user-images.githubusercontent.com/40791275/60663183-2e78ce80-9e67-11e9-8fc8-4384f5dedeff.png)
